site/content/docs/2.0/components/card.md

```diff
@@ -489,6 +489,27 @@ Borderless cards both remove the border and make the entire card clickable. You
 </div>
 {{< /example >}}
 
+### Title on Image
+
+Cards with the title on image style put the title of the card on top of the image with a gradient background. You can use any of the available utility classes to style your card, style its text, make it clickable, and change hover effects.
+
+{{< example >}}
+<div class="card bg-white shadow hover" style="max-width: 18rem;">
+  <div class="position-relative overflow-hidden">
+    <img class="hover-img-zoom-in card-img" src="{{< docsref `/assets/img/photo-gallery-demo/gallery-img-2.jpg` >}}" alt="University of Arizona Spring Fling" title="">
```

Comment:
```suggestion
    <img class="hover-img-zoom-in card-img" src="{{< docsref `/assets/img/photo-gallery-demo/gallery-img-2.jpg` >}}" alt="University of Arizona Spring Fling" title=""/>
```

Reply:
Not strictly required for HTML5, you can leave it off, but I was seeing weird indentation on the review site, and thought this might fix it, though it appears to be gone now.

Reply:
I now see there was a commit changing tabs to spaces, that existed in the code diff, but not the review site at the time that I reviewed.